### PR TITLE
Updating dependencies

### PR DESCRIPTION
**Pull request by Darylgolden**
## Overview / Explanation for Changes
Updated dependencies via poetry
## Oneline Summary of Changes

```
- Added new feature ... / Fixed bug ... / ... (:pr:`PR NUMBER HERE`)
```

## Acknowledgements
- [ ] I have read the [Contributing Guidelines](https://docs.manim.community/en/latest/contributing.html)

## Reviewer Checklist
- [ ] Newly added functions/classes are either private or have a docstring
- [ ] Newly added functions/classes have [tests](https://github.com/ManimCommunity/manim/wiki/Testing) added and (optional) examples in the docs
- [ ] Newly added documentation builds, looks correctly formatted, and adds no additional build warnings
- [ ] The oneline summary has been included [in the wiki](https://github.com/ManimCommunity/manim/wiki/Changelog-for-next-release)

### CODE REVIEW

**Comment by kolibril13** [comment]
Thanks for the pr!
Is there a reason that all optional parameters changes from true to false in poetry?

Further, there is an example notebook which I will delete (maybe pushed by accident?)

File: Untitled.ipynb [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+   "mimetype": "text/x-python",
+   "name": "python",
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.2"
+  }
+ },
+ "nbformat": 4,
+ "nbformat_minor": 5
+}
```
Comment (kolibril13) [inline Untitled.ipynb]:
```suggestion
```

**Comment by Darylgolden** [comment]
> Is there a reason that all optional parameters changes from true to false in poetry?

Not sure, I just did `poetry add ipywidgets` and poetry changed it on its own

**Comment by kolibril13** [comment]
ok

**Pull request merged**